CONTRIBUTING.md

```diff
@@ -75,7 +75,7 @@ Once you have Ruby, install [Bundler](https://bundler.io) with `gem install bund
 
 Get [Homebrew](https://brew.sh/) if you don't already have it.
 
-Install Watchman with `brew install watchman`, then install Java 8 with `brew tap caskroom/versions && brew cask install java8`.
+Install Watchman with `brew install watchman`, then install Java 8 with `brew cask install adoptopenjdk/openjdk/adoptopenjdk8`.
```

Comment:
Oracle Java 8, 9 and 10 are no longer available for public download due to a license change.